tests/components/autocomplete_dropdown.test.ts

```diff
@@ -39,33 +39,42 @@ async function typeInComposer(text: string, fromScratch: boolean = true) {
   return composerEl;
 }
 
-beforeEach(() => {
+const genericBeforeEach = () => {
```

Comment:
I'm not sure what the purpose of this function is if it's called only one, in a `beforeEach` :thinking: